cli/src/main.rs

```diff
@@ -69,71 +71,86 @@ enum Command {
 struct Opt {
     #[structopt(subcommand)]
     command: Command,
+
+    /// Whether to print cmac
+    #[structopt(short, long)]
+    cmac_flag: bool,
```

Comment:
This flag should belong to the encrypt/decrypt sub-commands. Please define it in the `EncryptDecryptOpt` struct.